source/android/Foundational_Components_Kernel.rst

```diff
@@ -132,30 +132,130 @@ new drivers should always be added as **modules**.
 
 To enable new modules:
 
-#. Run ``menuconfig`` as documented previously, Select ``=m`` for the driver.
+   #. Run ``menuconfig`` as documented previously, Select ``=m`` for the driver.
 
-#. Edit :file:`${YOUR_PATH}/ti-kernel-aosp/BUILD.bazel` to add your new module.
-   Look for the following section:
+   #. Edit :file:`${YOUR_PATH}/ti-kernel-aosp/BUILD.bazel` to add your new module.
+      Look for the following section:
 
-   .. code-block:: bash
+      .. code-block:: bash
```

Comment:
```suggestion
      .. code-block:: bazel
```
https://pygments.org/docs/lexers/#pygments.lexers.python.PythonLexer

Reply:
Fixed in v2 by adding a new commit.